satpy/dataset/metadata.py

```diff
@@ -27,33 +27,37 @@
 from satpy.writers.utils import flatten_dict
 
 
-def combine_metadata(*metadata_objects, average_times=True):
```

Comment:
Do we need a deprecation path, where a warning is raised if code passes `average_times`?

Reply:
I'm not sure any people are actually using this, but given that it exists means we thought someone might want to control it so I agree that it should be documented at the very least. A specific deprecation warning would be nice to have.

The changes in the multiscene code are also backwards incompatible, but very very unlikely to be used by anyone except maybe Adam and Ernst. If I remember correctly the default behavior is preserved and was changed when the related kwarg was added to the multiscene stacking function. So my vote is no deprecation warning on the multiscene stuff, but warning on the metadata.py `average_times` would be nice to have.

Reply:
I'll see about the deprecation warning, hopefully tomorrow.

Reply:
I added a `UserWarning` if someone tries to use the `average_times` kwarg.